src/main/java/io/confluent/connect/jdbc/util/QuoteMethod.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.connect.jdbc.util;
+
+public enum QuoteMethod {
+  ALWAYS("always"),
+  NEVER("never");
```

Comment:
Do we need the `name` members?  It seems we could omit the `name` and just use `toString().toLowerCase()`

Reply:
Yes, but then when using the default we also would have to use `toLowerCase(Local.ROOT)`. Seemed easier and less code this way.